Upgrade core-js to v3.x

**Description**
After upgrading to Babel v7.x, we experienced conflicts (with sub-dependencies) while importing `core-js` v3.x forcing us to use the latest v2.x branch. We should resolve these conflicts and upgrade `core-js` to v3.x using the following imports:

```
import "core-js/stable";
import "regenerator-runtime/runtime";
```
ref: https://github.com/zloirock/core-js#babel

Currently, we are importing polyfills for es5, es6, es7 as documented in v2.x.x, but I anticipate this will create problems in IE when we implement ES2017+ features.

There is an issue related to `simplebar` during the build where `Module not found: Error: Can't resolve...` multiple polyfills which prevents a working build. An issue for upgrading `simplebar` from `v3` to `v4` may help address this #1956 

```
ERROR in ./node_modules/simplebar/dist/simplebar.esm.js
Module not found: Error: Can't resolve 'core-js/modules/es6.array.from' in 'C:\Users\eplato\REPOS\reduxmerge\node_modules\simplebar\dist'
 @ ./node_modules/simplebar/dist/simplebar.esm.js 13:0-40
 @ ./web/js/components/util/scrollbar.js
 @ ./web/js/containers/sidebar/data.js
 @ ./web/js/containers/sidebar/sidebar.js
 @ ./web/js/app.js
 @ ./web/js/main.js

ERROR in ./node_modules/simplebar/dist/simplebar.esm.js
Module not found: Error: Can't resolve 'core-js/modules/es6.function.name' in 'C:\Users\eplato\REPOS\reduxmerge\node_modules\simplebar\dist'
 @ ./node_modules/simplebar/dist/simplebar.esm.js 10:0-43
 @ ./web/js/components/util/scrollbar.js
 @ ./web/js/containers/sidebar/data.js
 @ ./web/js/containers/sidebar/sidebar.js
 @ ./web/js/app.js
 @ ./web/js/main.js

ERROR in ./node_modules/simplebar/dist/simplebar.esm.js
Module not found: Error: Can't resolve 'core-js/modules/es6.object.assign' in 'C:\Users\eplato\REPOS\reduxmerge\node_modules\simplebar\dist'
 @ ./node_modules/simplebar/dist/simplebar.esm.js 14:0-43
 @ ./web/js/components/util/scrollbar.js
 @ ./web/js/containers/sidebar/data.js
 @ ./web/js/containers/sidebar/sidebar.js
 @ ./web/js/app.js
 @ ./web/js/main.js

ERROR in ./node_modules/simplebar/dist/simplebar.esm.js
Module not found: Error: Can't resolve 'core-js/modules/es6.regexp.match' in 'C:\Users\eplato\REPOS\reduxmerge\node_modules\simplebar\dist'
 @ ./node_modules/simplebar/dist/simplebar.esm.js 11:0-42
 @ ./web/js/components/util/scrollbar.js
 @ ./web/js/containers/sidebar/data.js
 @ ./web/js/containers/sidebar/sidebar.js
 @ ./web/js/app.js
 @ ./web/js/main.js

ERROR in ./node_modules/simplebar/dist/simplebar.esm.js
Module not found: Error: Can't resolve 'core-js/modules/es6.regexp.replace' in 'C:\Users\eplato\REPOS\reduxmerge\node_modules\simplebar\dist'
 @ ./node_modules/simplebar/dist/simplebar.esm.js 9:0-44
 @ ./web/js/components/util/scrollbar.js
 @ ./web/js/containers/sidebar/data.js
 @ ./web/js/containers/sidebar/sidebar.js
 @ ./web/js/app.js
 @ ./web/js/main.js

ERROR in ./node_modules/simplebar/dist/simplebar.esm.js
Module not found: Error: Can't resolve 'core-js/modules/web.dom.iterable' in 'C:\Users\eplato\REPOS\reduxmerge\node_modules\simplebar\dist'
 @ ./node_modules/simplebar/dist/simplebar.esm.js 12:0-42
 @ ./web/js/components/util/scrollbar.js
 @ ./web/js/containers/sidebar/data.js
 @ ./web/js/containers/sidebar/sidebar.js
 @ ./web/js/app.js
 @ ./web/js/main.js
```

It appears others (`https://github.com/zloirock/core-js/issues/500`) have encountered this `Module not found: Error: Can't resolve...` issue and have recommended downgrading to `core-js` v2 which we currently are using to build.

--- a/package.json
+++ b/package.json
@@ -87,7 +87,7 @@
     "cheerio": "^1.0.0-rc.2",
     "chromedriver": "^77.0.0",
     "clean-webpack-plugin": "^3.0.0",
-    "core-js": "^2.6.9",
+    "core-js": "^3.3.4",
     "cross-env": "^6.0.2",
     "css-hot-loader": "^1.4.1",
     "css-loader": "^3.2.0",

--- a/web/js/main.js
+++ b/web/js/main.js
@@ -1,8 +1,6 @@
 /* global DEBUG */
 // IE11 corejs polyfills container
-import 'core-js/es5';
-import 'core-js/es6';
-import 'core-js/es7';
+import 'core-js/stable';
 import 'regenerator-runtime/runtime';
 // IE11 corejs polyfills container
 import 'whatwg-fetch';

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -137,7 +137,7 @@ module.exports = {
             compact: false // fixes https://stackoverflow.com/questions/29576341/what-does-the-code-generator-has-deoptimised-the-styling-of-some-file-as-it-e
           }
         },
-        exclude: [/\.test\.js$/, /fixtures\.js$/]
+        exclude: [/\.test\.js$/, /fixtures\.js$/, /core-js/]
       },
       {
         test: require.resolve('jquery'), // expose globally for jQuery plugins

